daffodil-test/src/test/resources/org/apache/daffodil/section06/namespaces/namespaces.tdml

```diff
@@ -2415,4 +2417,23 @@
     </tdml:infoset>
   </tdml:parserTestCase>
 
+  <!--
+    Test name: schemaNoGlobalElems_03
+    Schema: multi_base_21.dfdl.xsd
+    Root: base2
+    Purpose: This test demonstrates that you can have a schema if there are
+             no global elements in the primary schema, as long as the root element is
+             specified (the TDML Runner always specifies the root)
+  -->
+
+  <tdml:parserTestCase name="schemaNoGlobalElems_03"
+    root="distant_element" model="multi_base_21.dfdl.xsd" description="">
```

Comment:
Therefore, the root="distant_element" attribute here is redundant since the test case has an infoset with the distant_element as well.

Reply:
TDML runner used to require the root attribute. When we made it smarter to look at the infoset, we decided not to make it illegal to have the root element, but to check consistency if it is present. Just to avoid editing hundreds of tests for a non-functional reason.

Reply:
Ah yeah, I missed this comment. I do like the consistency check.